web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -1,9 +1,7 @@
 import React from 'react';
 import ReactHtmlParser from 'react-html-parser';
 import { makeStyles } from '@mui/styles';
-import Grid from '@mui/material/Grid';
-// import Box from '@mui/material/Box';
-import Link from '@mui/material/Link';
+import { Box, Grid, Link } from '@mui/material';
```

Comment:
MUI 5 supports tree shaking by default now, so there shouldn't be issue using named imports